.github/workflows/dotnet.yml

```
@@ -0,0 +1,43 @@
+      - main
+      - releases/*
+    paths-ignore:
+      - '**.md'
+      - 'applications/rpg_table_helper/**'
+
+jobs:
+  build:
+    name: Build & Test Solution
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
```

yamllint

[error] wrong indentation: expected 6 but found 4

(indentation)